ProjectLighthouse/Helpers/VersionHelper.cs

```diff
@@ -48,13 +51,16 @@ static VersionHelper()
                 "Please make sure you are properly disclosing the source code to any users who may be using this instance.",
                 LogArea.Startup
             );
+            Revision = $"{Branch}-dirty_r{rawRevision}";
             CanCheckForUpdates = false;
         }
     }
 
     public static string CommitHash { get; set; }
     public static string Branch { get; set; }
-    public static string FullVersion => $"Project Lighthouse {Branch}@{CommitHash} {Build} ({ServerConfiguration.Instance.Customization.ServerName})";
+    public static string Revision { get; set; }
+    public static string EnvVer => $"{ServerConfiguration.Instance.Customization.EnvironmentName} {Revision}";
+    public static string FullVersion => $"Project Lighthouse {Branch}@{CommitHash} {Build} ({EnvVer})";
```

Comment:
Including `EnvVer` here causes redundant info to display. The old way is better IMO:

```suggestion
    public static string FullVersion => $"Project Lighthouse {Branch}@{CommitHash} {Build} ({ServerConfiguration.Instance.Customization.ServerName})";
```

Reply:
It displaying the envname is probably a bit better as well, displaying the name is a bit redundant too (especially on the site when you're told it in the title and homepage's header)

but yeah, displaying the hash and build in the default mode is a bit redundant

Reply:
Honestly we might be fine without the brand at all